smart.json

```diff
@@ -1,9 +1,9 @@
 {
```

Comment:
If I remember correctly, the `smart.json` is for packages before the current Meteor package system. We could remove the file then.